### PR TITLE
Identity v3: Users Update

### PR DESCRIPTION
**Pull request by jtopjian**
For #287

### CODE REVIEW

File: openstack/identity/v3/users/testing/fixtures.go [hunk 1]
```diff
@@ -127,6 +127,41 @@ const CreateNoOptionsRequest = `
 }
 `
 
+// UpdateRequest provides the input to as Update request.
+const UpdateRequest = `
+{
+    "user": {
+        "enabled": false,
+        "disabled_reason": "DDOS",
```
Comment (jrperritt) [inline openstack/identity/v3/users/testing/fixtures.go]: :+1:

File: openstack/identity/v3/users/testing/fixtures.go [hunk 2]
```diff
@@ -183,6 +218,26 @@ var SecondUserNoOptions = users.User{
...
+	Enabled:          false,
+	ID:               "9fe1d3",
+	Links: map[string]interface{}{
+		"self": "https://example.com/identity/v3/users/9fe1d3",
+	},
+	Name:              "jsmith",
+	PasswordExpiresAt: SecondUserPasswordExpiresAt,
+	Extra: map[string]interface{}{
+		"email":           "jsmith@example.com",
+		"disabled_reason": "DDOS",
```
Comment (jrperritt) [inline openstack/identity/v3/users/testing/fixtures.go]: :+1:

**Comment by jrperritt** [comment]
Looks good

**Comment by jtopjian** [comment]
Thanks for pushing these through :)

**Pull request merged**